<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_algorithm_settings</class>
 <widget class="QDialog" name="dialog_algorithm_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>540</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Surrogates</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Learning method</string>
    </property>
    <property name="buddy">
     <cstring>comboBox_learningmethod</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_learningmethod">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of training data</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_nbtrainingdata</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_nbtrainingdata">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Logger</string>
   </property>
   <widget class="QRadioButton" name="radioButton_logger_summary">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Summary</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_logger_detailed">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Detailed</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Algorithm Run</string>
   </property>
   <widget class="QRadioButton" name="radioButton_algorithmrun_standard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Standard</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_algorithmrun_analyseperformance">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyse performance</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>451</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Algorithm</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabu list length</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_tabulistlength</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_tabulistlength">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>35</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_nbreplicationsmengeincremental">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of replications of Menge per analysis during tabu search</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_nbreplicationsmengeincremental</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resetvaluesTS">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset values</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum number of candidate solutions evaluated per iteration</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_max_candidates_per_iteration</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_max_candidates_per_iteration">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Replication budget tabu search</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_budget_tabusearch</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_budget_tabusearch">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Replication budget identification step</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_budget_identificationstep</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_budget_identificationstep">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialog_algorithm_settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialog_algorithm_settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
